.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,18 @@
 <!--  Thanks for sending a pull request!  Here are some tips for you:
 1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+2. Please label this pull request according to what type of issue you are fixing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://github.com/kubernetes/community/blob/master/contributors/devel/release.md#issue-kind-label
+3. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+4. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
+5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
 -->
 
+**What type of PR is this?**
+> Uncomment only one, leave it on its own line:
+>
+> /kind bug
```

Comment:
Yes, I was wondering if it was perhaps too laborious for folks to choose from a long list of labels (esp since the issue template doesn't list them) but I'm happy to add them all.